src/version/sagas/feedsListSagas.js

```diff
@@ -0,0 +1,23 @@
+import { takeEvery, put } from 'redux-saga/effects';
+
+import { token, domainName } from "../../core/token";
+import { FEEDS_LIST_ACTION, feedsListAction } from "../actions/feedsListAction";
+
+export default takeEvery(FEEDS_LIST_ACTION.REQUEST, function*(action) {
+    const apiPatientsUser = 'api/feeds';
```

Comment:
I don't understand why you create this const. It's only used once, and just after its creation...

Reply:
Done